scripts/battle/card/Card.cs

```diff
@@ -0,0 +1,156 @@
+using Godot;
+using System.Collections.Generic;
+
+public partial class Card : Node2D {
+	[Signal]
+	public delegate void CardSelectedEventHandler(Card card);
+	public Color color { get; set; }
+	public Button selectButton;
+	
+	
+	public string Id { get; set;} // card_id
+
+	public TargetEnum Target { get; set;}
```

Comment:
This is handled by the Battle code and doesn't need to be saved by the card.

Reply:
done